doc/endpoint/statuscodes.md

```diff
@@ -51,7 +56,12 @@ Sometime at runtime status mapping resolution can not work properly because of t
 For example this code will fail at compile time; because of type erasure `Right[NotFound]` and `Right[BadRequest]` will 
 become `Right[Any]`, therefore the code would not be able to find the correct mapping for a value:
 
-```scala
+```scala mdoc:fail
```

Comment:
why `fail`? :)

Reply:
ah ok, it's a doesn't-compile example

Reply:
Indeed 😄 . This is how it looks like, the compilation error is even included:

![Screenshot 2020-08-07 at 14 42 31](https://user-images.githubusercontent.com/132215/89646673-7ebd2e00-d8bc-11ea-8a38-fed80619d848.png)

Reply:
nice! :)